frontend/src/scenes/actions/actionEditLogic.ts

```diff
@@ -35,8 +39,14 @@ export const actionEditLogic = kea<actionEditLogicType>([
         }),
         setCreateNew: (createNew: boolean) => ({ createNew }),
         actionAlreadyExists: (actionId: number | null) => ({ actionId }),
+        deleteAction: true,
```

Comment:
Is there an eventual goal of renaming the variables too to something else? Or we need to make sure it's communicated somewhere what actions are

Reply:
Ideally our code would reflect naming in the UI, but I don't think this refactor is recommended since so many things in our code have action in them. Perhaps a compromise could be renaming the folder these `action*` files are in to events or something.